### PR TITLE
Add test for Default VPC

### PR DESCRIPTION
**Pull request by gargravarr**
Since I was already coding a test, let's increase code coverage.

Signed-off-by: rreis <rodrigo@zalando.de>

### CODE REVIEW

**Comment by gargravarr** [comment]
:+1:

File: provisioner/aws_test.go [hunk 1]
```diff
@@ -251,6 +267,14 @@ func TestGetStackByName(t *testing.T) {
 	}
 }
 
+func TestGetDefaultVPC(t *testing.T) {
+	a := newAWSAdapterWithStubs("", "GroupName")
+	_, err := a.GetDefaultVPC()
```
Comment (mikkeloscar) [inline provisioner/aws_test.go:272]: maybe also check that the vpc returned here is not `nil`.

**Comment by gargravarr** [comment]
:+1:

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**